src/App.js

```diff
@@ -1,19 +1,107 @@
 import React from 'react';
+import TodoList from './components/TodoList/TodoList';
 import './App.css';
 
 import users from './api/users';
+import todoList from './api/todos';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+class App extends React.Component {
+  state = {
+    todos: todoList,
+    select: '',
+    input: '',
+    errorTodo: false,
+    errorUser: false,
+  };
+
+  handleChangeInput = (event) => {
+    this.setState({
+      input: event.target.value,
+      errorTodo: false,
+    });
+  };
+
+  handleChangeSelect = (event) => {
+    this.setState({
+      select: event.target.value,
+      errorUser: false,
+    });
+  };
+
+  formSubmit = (event) => {
+    event.preventDefault();
+
+    if (this.state.input === '' || this.state.select === '') {
+      this.setState(prevState => ({
+        errorTodo: prevState.input === '',
+        errorUser: prevState.select === '',
+      }));
+
+      return;
+    }
+
+    this.setState(prevState => ({
```

Comment:
Тут тоже можно делать деструктуризацию 
```suggestion
    this.setState(({ todos, select, input })=> ({
```